timescaledb/quick-start/node.md

```diff
@@ -157,6 +157,11 @@ Connection has been established successfully.
 Now that we have a successful connection to the `defaultdb` database, we
 can build out our first database and model.
 
+<highlight type="warning">
+You can skip the steps 1 and 2 if you're going to use TimescaleDB cloud. The
```

Comment:
Link to either the Cloud sign-up page or the Docs "cloud getting started" page. @Loquacity or @lukas-bernert have thoughts?

Reply:
Point to the new cloud install page: timescaledb/:currentVersion:/how-to-guides/install-timescaledb/installation-cloud/

After https://github.com/timescale/docs/pull/583/ merges, that page will go live, note that we will be moving it again shortly, though.